haiku-apps/helios/helios-1.7.2.recipe

```diff
@@ -3,11 +3,11 @@ DESCRIPTION="Cdrecord GUI frontend."
 HOMEPAGE="https://github.com/HaikuArchives/Helios"
 COPYRIGHT="2000-2004 Maurice Michalski"
 LICENSE="MIT"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="git+https://github.com/HaikuArchives/Helios.git#ffbbca7"
```

Comment:
You should just move this to a later commit instead. Looks like most x86_64 issues has been fixed upstream. See https://github.com/HaikuArchives/Helios/commit/b870e477789928027dd4f3a444219cb820944678